src/app/workflow/info-tab/info-tab.component.ts

```diff
@@ -119,6 +120,7 @@ export class InfoTabComponent extends EntryTab implements OnInit, OnChanges {
         this.selectedVersion.workflow_path,
         this.sessionQuery.getValue().entryType
       );
+      this.sourceCodeFile = this.selectedVersion.workflow_path.slice(1); //removes first slash to prevent from displaying on text
```

Comment:
I just queried the workflowversions table in  recent-ish snapshot of prod data  for workflowpath not starting with a `/`, and about 1% of them don't -- this would end up truncating the first character in those cases. So I think you need either need to check here for a leading slash, or just take the whole value here and change the display logic in the HTML.

I ran this query:

```
select count(*) from workflowversion where workflowpath not like '/%';
```